packages/tailwindcss/src/utils/infer-data-type.ts

```diff
@@ -283,3 +287,38 @@ function isBackgroundSize(value: string) {
 
   return count > 0
 }
+
+/* -------------------------------------------------------------------------- */
+
+/**
+ * Determine if `value` is valid angle
+ *
+ * <angle> = <number><units>
+ * <units> = deg | rad | grad | turn
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/CSS/angle
+ */
```

Comment:
this comment should be attached to the function — for consistency with some of the others and so it shows up in documentation popups / hovers.